test_data/asm/failing_assertion.asm

```diff
@@ -1,9 +1,10 @@
 use std::check;
 
 machine Empty {
-    let line: col = |i| i - 7;
+    let line = |i| i - 7;
+    let line_col: col = line;
```

Comment:
So the previous case is not valid anymore?

Reply:
No, unfortunately not.